python3.11-venv-pyd4/Dockerfile

```diff
@@ -21,6 +21,7 @@ ENV PATH="/root/.cargo/bin:${PATH}"
 RUN groupadd --gid 1000 worker && useradd -g worker --uid 1000 --shell /usr/sbin/nologin --create-home worker
 
 # Install d4tools
+RUN echo 'location' >> ~/.curlrc # https://github.com/38/d4-format/pull/77#issuecomment-2044438359
```

Comment:
I fixed this the same as I fixed the GitHub action in chanjo2. This is due to a bug in d4tools that has not yet fixed  (https://github.com/38/d4-format/pull/77)